loopy/options.py

```diff
@@ -210,7 +211,9 @@ def __init__(
                 trace_assignments=kwargs.get("trace_assignments", False),
                 trace_assignment_values=kwargs.get("trace_assignment_values", False),
 
-                skip_arg_checks=kwargs.get("skip_arg_checks", sys.flags.optimize),
+                skip_arg_checks=kwargs.get("skip_arg_checks",
+                    sys.flags.optimize
+                    or bool(os.environ.get("_LOOPY_SKIP_ARG_CHECKS"))),
```

Comment:
if optimize is on, but _LOOPY_SKIP_ARG_CHECKS=0, this still skips the arg checks, right? Is that the intention you had?

Reply:
That's fair. It was not my intention to make `_LOOPY_SKIP_ARG_CHECKS` into documented behavior though, so I'm OK tolerating this behavioral weirdness... let's say the rule is just if `_LOOPY_SKIP_ARG_CHECKS` is set to something non-empty, it's considered set. It's just a hack that's needed because `pytest` is incompatible with `python -O`. It relies on `assert` doing something, after all.

Reply:
2763e3e2